core/src/main/java/org/infinispan/interceptors/distribution/TxDistributionInterceptor.java

```diff
@@ -321,8 +326,8 @@ private Object handleSecondPhaseCommand(TxInvocationContext ctx, TransactionBoun
    }
 
    protected void checkTxCommandResponses(Map<Address, Response> responseMap,
-         TransactionBoundaryCommand command, TxInvocationContext<LocalTransaction> context,
-         Collection<Address> recipients) {
+                                          TransactionBoundaryCommand command, TxInvocationContext<LocalTransaction> context,
```

Comment:
Could you please set up your IDE to not autoformat lines that you are not changing?

Reply:
ok. When I apply the infinispan formatter, sometimes formats lines that are not in the formatter way. I didn't know I could say (without just choosing myself the lines that I want to format) just to format modified lines.

Reply:
What do you mean by "apply the infinispan formatter"? Don't do that if it "fixes" too much - you can also format just selected text.

Reply:
I don't do it all the time, but sometimes it happens by mistake. I'm going to see the changes I did too much for this case and repush